sources/osgShader/node/shadows.js

```diff
@@ -13,7 +13,7 @@ define( [
 
     ShadowReceive.prototype = MACROUTILS.objectInherit( Node.prototype, {
         type: 'ShadowReceiveNode',
-        validInputs: [ 'lighted', 'shadowTexture', 'shadowTextureMapSize', 'shadowTextureProjectionMatrix', 'shadowTextureViewMatrix', 'shadowTextureDepthRange', 'lightNDL', 'vertexWorld', 'shadowbias' /* 'shadowexponent0', 'shadowexponent1', 'shadowepsilonVSM' */ ],
+        validInputs: [ 'lighted', 'shadowTexture', 'shadowTextureMapSize', 'shadowTextureProjectionMatrix', 'shadowTextureViewMatrix', 'shadowTextureDepthRange', 'lightNDL', 'vertexWorld', 'shadowbias', /* 'shadowexponent0', 'shadowexponent1', 'shadowepsilonVSM' */ ],
```

Comment:
it's used but not mandatory (EVSM/VSM/ESM only), that's the idea of the comment.